tests/integration/models/test_preferences.py

```diff
@@ -0,0 +1,83 @@
+import mock
+
+from praw.models import Preferences
+from .. import IntegrationTest
+
+
+class TestPreferences(IntegrationTest):
+    def test_creation(self):
+        prefs_obj = self.reddit.user.preferences
+        assert isinstance(prefs_obj, Preferences)
+
+    def test_view(self):
+        some_known_keys = ('allow_clicktracking', 'default_comment_sort',
```

Comment:
Consider making this a `set`, and then using `some_known_keys.issubset(prefs_dict)` instead of the for loop.